### DISCUSSION TITLE
Round robin on key-down as an option, rather than on key-up.

### DISCUSSION THREAD

**Discussion by nick-hopoo** [OpenMultiBoxing/OpenMultiBoxing#5] [Ideas]
The goal would be to allow players to quickly input something like CTRL+1 and have it transmit to another window, because LCONTROL was held down, rather than having to tap it and then press the desired key input.

This way you could rebind your secondary clients and transmit commands directly to them smoothly, and still entirely manually.

**Comment by mooreatv** [comment]
With version 7 having full broadcasting, do you think that's still needed (vs using various modifiers/bindings in windows)?
